### ISSUE TITLE
Hide two sections of a11y errors

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#540]
On the [a11y agencies page](https://staging.pulse.cio.gov/accessibility/agencies/), hide the `Form Errors` and `Other Errors` column.

**Comment by gbinal** [comment]
This is addressed by https://github.com/18F/pulse/pull/549

**Comment by gbinal** [comment]
Done.

**Comment by gbinal** [comment]
Actually, I spoke too soon.  This information is also represented on the domains page and on the individual errors page, so I assume they need to be hidden as well.

**Comment by gbinal** [comment]
For the domain page, there's no good way to remove the info only from the presentation layer.  Instead, I need to remove that datapoint from the `domains.json` file or change all of its values to `0`.  Note though that the total errors would need to be adjusted in the later case.  It's a workable option though.  

I should note the key file locations and weigh asking for help commenting them out in the presentation layer instead of hacking it myself.

**Comment by gbinal** [comment]
Note that from [earlier](https://github.com/18F/pulse/issues/394#issuecomment-203001988), it would seem that `Form` errors = `1_3_1 (missing label) 1_3_1(missing legend)` and that `Other Errors` is everything else that isn't 
`1_4(contrast error); 1_1(missing alt); 4_1_2(missing anchor), 4_1_1(duplicate id); 1_3_1 (missing label) 1_3_1(missing legend)`.

**Comment by gbinal** [comment]
I ran a full scan and distilled the codes of the resulting errors to add more to the `ignore` list.  

````
WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG3
WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG4
WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG5
WCAG2AA.Principle1.Guideline1_1.1_1_1.H2.EG5
WCAG2AA.Principle1.Guideline1_1.1_1_1.H24
WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2
WCAG2AA.Principle1.Guideline1_1.1_1_1.H36
WCAG2AA.Principle1.Guideline1_1.1_1_1.H37
WCAG2AA.Principle1.Guideline1_1.1_1_1.H53
WCAG2AA.Principle1.Guideline1_1.1_1_1.H67.1
WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
WCAG2AA.Principle1.Guideline1_3.1_3_1.H39.3.LayoutTable
WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
WCAG2AA.Principle1.Guideline1_3.1_3_1.H43.HeadersRequired
WCAG2AA.Principle1.Guideline1_3.1_3_1.H71.NoLegend
WCAG2AA.Principle1.Guideline1_3.1_3_1.H73.3.LayoutTable
WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail
WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
WCAG2AA.Principle2.Guideline2_2.2_2_1.F41.2
WCAG2AA.Principle2.Guideline2_4.2_4_1.G1,G123,G124.NoSuchID
WCAG2AA.Principle2.Guideline2_4.2_4_1.H64.1
WCAG2AA.Principle2.Guideline2_4.2_4_2.H25.1.EmptyTitle
WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.3.Lang
WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.3.XmlLang
WCAG2AA.Principle3.Guideline3_1.3_1_2.H58.1.Lang
WCAG2AA.Principle3.Guideline3_2.3_2_2.H32.2
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.NoContent
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Button.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Fieldset.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputButton.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputCheckbox.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputImage.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputPassword.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputRadio.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.InputText.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Textarea.Name
````

**Comment by gbinal** [comment]
Hopefully https://github.com/18F/domain-scan/pull/87 addresses this.

**Comment by gbinal** [comment]
Done.
